frontend/app/src/components/calendar/CalendarNavigator.vue

```diff
@@ -0,0 +1,91 @@
+<script setup lang="ts">
+import dayjs, { type Dayjs } from 'dayjs';
+import { DateFormat } from '@/types/date-format';
+
+const props = defineProps<{
+  value: Dayjs;
+}>();
+
+const emit = defineEmits<{
+  (e: 'input', date: Dayjs): void;
+}>();
+
+const vModel = useSimpleVModel(props, emit);
+
+function prevMonth() {
+  const prevMonth = get(vModel).subtract(1, 'month');
+  set(vModel, prevMonth);
+}
+
+function nextMonth() {
+  const nextMonth = get(vModel).add(1, 'month');
+  set(vModel, nextMonth);
+}
+
+const readableMonthAndYear = computed(() => get(vModel).format('MMMM YYYY'));
+
+const datetime = ref<string>('');
+
+watchImmediate(vModel, (value) => {
+  set(datetime, convertFromTimestamp(value.unix()));
+});
+
+function input(value: string) {
+  const timestamp = convertToTimestamp(
+    value,
+    DateFormat.DateMonthYearHourMinuteSecond,
+    true,
+  );
+
+  set(vModel, dayjs(timestamp));
+}
```

Comment:
do you think we can wrap this to a get/set computed to pass a v-model to the DateTimePicker instead of using value/input?